content/docs/start/experiments.md

```diff
@@ -4,237 +4,227 @@ title: 'Get Started: Experiments'
 
-DVC makes it easy to iterate on your project using Git commits, tags, or
-branches. You can try different ideas quickly by tuning
-[parameters](/doc/command-reference/params), compare their performance with
-[metrics](/doc/command-reference/metrics), and visualize them with
-[plots](/doc/command-reference/plots).
+⚠️ This feature is only available in DVC 2.0 ⚠️
 
-Read on or watch our video to see how it's done!
-
-https://youtu.be/iduHPtBncBk
+Experiments proliferate quickly in ML projects where there are many parameters
+to tune or other permutations of the code or data. We can organize such projects
+and only keep what we ultimately need with
+[experiments](/doc/command-reference/exp).
 
 > 📖 See [Experiment Management](/doc/user-guide/experiment-management) for more
 > information on DVC's approach.
 
-## Collecting metrics
+## Using experiments
+
+In the previous section, we learned how to tune
+[ML pipelines](/doc/tutorials/get-started/ml-pipeline) and compare the changes.
+Let's further increase the number of features in the `featurize` stage to see
+how it compares.
 
-First, let's see what is the mechanism to capture values for these ML experiment
-attributes. Let's add a final evaluation stage to our
-[pipeline](/doc/tutorials/get-started/data-pipelines#dependency-graphs-dags):
+`dvc exp run` makes it even easier to try a new experiment:
 
 ```dvc
-$ dvc run -n evaluate \
-          -d src/evaluate.py -d model.pkl -d data/features \
-          -M scores.json \
-          --plots-no-cache prc.json \
-          python src/evaluate.py model.pkl \
-                 data/features scores.json prc.json
+$ dvc exp run --set-param featurize.max_features=3000
 ```
 
 <details>
 
-### 💡 Expand to see what happens under the hood.
+### 💡 Expand to see what this command does.
 
-The `-M` option here specifies a metrics file, while `--plots-no-cache`
-specifies a plots file produced by this stage that will not be
-<abbr>cached</abbr> by DVC. `dvc run` generates a new stage in the `dvc.yaml`
-file:
+`dvc exp run` is similar to `dvc repro` but with some added conveniences for
+running experiments. The `--set-param` (or `-S`) flag sets the values for
+[parameters](/doc/command-reference/params) as a shortcut to editing
+`params.yaml`.
 
-```yaml
-evaluate:
-  cmd: python src/evaluate.py model.pkl data/features ...
-  deps:
-    - data/features
-    - model.pkl
-    - src/evaluate.py
-  metrics:
-    - scores.json:
-        cache: false
-  plots:
-    - prc.json:
-        cache: false
-```
+Check that the `featurize.max_features` value has been updated in `params.yaml`:
 
-The biggest difference to previous stages in our pipeline is in two new
-sections: `metrics` and `plots`. These are used to mark certain files containing
-experiment "telemetry". Metrics files contain simple numeric values (e.g. `AUC`)
-and plots files contain matrices and data series (e.g. `ROC` or model loss
-plots) that are meant to be visualized and compared.
+```diff
```

Comment:
Hmm, this was copied from all the other experiments pages, although I think you are right that `git` might look slightly better (it colors the entire line instead of coloring only the plus/minus signs). Let me know if you have a preference here:
1. Leave as is
2. Change this one instance
3. Change all instances (here or in a separate PR)

Reply:
> this was copied from all the other experiments pages

Shoot, I see it's in 3 other misc. docs indeed. I'll update those separately 🙂 so my vote goes to # 2 here.

Reply:
I see you already went for `git`. Resolving